### DISCUSSION TITLE
Editor: working example of implementing Quill modules

### DISCUSSION THREAD

**Discussion by pimpmypixel** [primefaces/community#1910] [PrimeReact]
I have been looking for a working example of Quill with loaded modules (quill-html-edit-button, quill-image-uploader), but have so far not been able to find any. Are we discouraged from using modules in general or are they just not playing well with the Prime version of Quill?

![image](https://github.com/primefaces/community/assets/15387119/05040ceb-0510-4187-8f40-32ecbb7c6304)

Many thanks to anyone who can point me in the right direction 🙏

**Comment by nitrogenous** [comment]
I apologize for any confusion. Could you please provide more details about your question? Additionally, could you clarify in what context you are referring to PrimeReact? This will help us provide a more accurate and helpful response

**Comment by pimpmypixel** [comment]
Hi! I’m just curious when you plan to release 10.7.0 of PrimeReact as it releases a fix for the Dropdown component <https://github.com/primefaces/primereact/issues/6591>.
